ONIT.VismaNet/Lib/VismaNetApiHelper.cs

```diff
@@ -269,15 +269,16 @@ private static int TryParseToInt(string value)
             }
         }
 
-        internal static async Task<T> Create<T>(T entity, string apiControllerUri, VismaNetAuthorization authorization)
+        internal static async Task<T> Create<T>(T entity, string apiControllerUri, VismaNetAuthorization authorization, string apiUriToGetFrom=null)
             where T : DtoProviderBase
         {
             using (var webclient = GetHttpClient(authorization))
             {
                 var apiUrl = GetApiUrlForController(apiControllerUri);
+                var apiGetUrl = GetApiUrlForController(apiUriToGetFrom);
```

Comment:
We should check if apiUriToGetFrom is NULL and use either or instead of try/catching.